src/middlewares/validateClaim.js

```diff
@@ -0,0 +1,102 @@
+const { checkSchema } = require('express-validator');
+
+const { claimTypes } = require('../database/constants');
+const { ERRORS } = require('../translates');
+
+module.exports = checkSchema({
+  date: {
+    in: ['body'],
+    custom: {
+      errorMessage: ERRORS.INVALID_DATE,
+      options: (value) => !Number.isNaN(Date.parse(value)),
+    },
+    exists: true,
+  },
+  type: {
+    in: ['body'],
+    isIn: {
+      options: [claimTypes],
+      errorMessage: ERRORS.INVALID_CLAIM_TYPE,
+    },
+    exists: true,
+  },
+  user_id: {
+    in: ['body'],
+    isLength: {
+      errorMessage: ERRORS.INVALID_USER_ID,
+      options: { min: 36, max: 36 },
```

Comment:
I've fixed check that it is UUID, but validation for user existing a bit tricky here because it's another service. Anyway it is validates in the next middleware.